src/Bicep.Core.UnitTests/TypeSystem/TypeValidatorAssignabilityTests.cs

```diff
@@ -997,7 +998,7 @@ private TypeSymbol CreateDummyResourceType()
         {
             var typeReference = ResourceTypeReference.Parse("Mock.Rp/mockType@2020-01-01");
 
-            return new ResourceType(typeReference, new NamedObjectType(typeReference.FormatName(), LanguageConstants.CreateResourceProperties(typeReference), null));
+            return new ResourceType(typeReference, new NamedObjectType(typeReference.FormatName(), TypeSymbolValidationFlags.Default, LanguageConstants.CreateResourceProperties(typeReference), null), TypeSymbolValidationFlags.Default);
```

Comment:
>Default [](start = 125, length = 7)

Are there any tests that would benefit from testing with Permissive as well?

Reply:
Added.